src/Provider/Provider.Web/ViewModels/ApplicationReview/ApplicationReviewViewModel.cs

```diff
@@ -33,6 +33,7 @@ public class ApplicationReviewViewModel : VacancyRouteModel
         public string Support { get; set; }
         public List<TrainingCoursesViewModel> TrainingCourses { get; set; }
         public List<WorkExperienceViewModel> WorkExperiences { get; set; }
+        public ApprenticeshipTypes ApprenticeshipType { get; set; }
```

Comment:
this should be nullable defaulted to standard

Reply:
Corrected in latest commit.